frontend/apps/reader/modules/readerview.lua

```diff
@@ -575,6 +641,25 @@ function ReaderView:drawHighlightRect(bb, _x, _y, rect, drawer)
     elseif drawer == "invert" then
         bb:invertRect(x, y, w, h)
     end
+    if is_note_mark then
+        if self.highlight.note_mark == "underline" then
+            bb:paintRect(x, y + h - 1, w, 2, Blitbuffer.COLOR_BLACK)
+        else
+            local note_mark_pos_x
+            if self.ui.paging or
+                    (self.ui.document:getVisiblePageCount() == 1) or -- one-page mode
+                    (x < Screen:getWidth() / 2) then -- page 1 in two-page mode
+                note_mark_pos_x = self.note_mark_pos_x1
+            else
+                note_mark_pos_x = self.note_mark_pos_x2
+            end
+            if self.highlight.note_mark == "sideline" then
+                bb:paintRect(note_mark_pos_x, y, 3, h, Blitbuffer.COLOR_BLACK)
```

Comment:
As this `3` should be the same value as the one mentionned above, may be put it in a module level variable, or a `self.note_mark_sideline_thickness` member ?